src/components/vm/vmActions.jsx

```diff
@@ -312,6 +314,27 @@ const VmActions = ({ vm, dispatch, storagePools, onAddErrorNotification, isDetai
         );
     }
 
+    if (vm.state !== "shut off") {
```

Comment:
OOI: If anything, it should be easier to migrate a  shut off VM than a running one. Is that some limitation of libvirt, or a design decision in c-machines for some reason? (would be nice to have a comment on that)

Reply:
As discussed, Live migration is is much bigger use case, to minimize downtime. Offline migration is nothing else, then just copying XML of shut off VM from one side to the other. And even for that, libvirt guys recommend to do it manually (`virsh dumpxml` and then on the other side `virsh define`), since that manual way allows more flexibility